src/performance/scala-2.11/markets/MarketActorBenchmarkSimulation.scala

```diff
@@ -0,0 +1,79 @@
+package markets
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.agent.Agent
+import akka.testkit.TestKit
+
+import com.typesafe.config.{Config, ConfigFactory}
+import markets.engines.CDAMatchingEngine
+import markets.orders.orderings.ask.AskPriceTimeOrdering
+import markets.orders.orderings.bid.BidPriceTimeOrdering
+import markets.participants.strategies.TestRandomTradingStrategy
+import markets.participants.{SubmitAskOrder, SubmitBidOrder}
+import markets.settlement.TestSettlementMechanismActor
+import markets.tickers.Tick
+import markets.tradables.{TestTradable, Tradable}
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+import scala.util.Random
+
+
+object MarketActorBenchmarkSimulation extends App {
+
+  val baseConfig = ConfigFactory.load("marketActorBenchmark.conf")
+  val akkaConfig = ConfigFactory.parseString(s"akka.actor.default-dispatcher.fork-join-executor.parallelism-max=${args(0)}")
+  val appConfig = akkaConfig.withFallback(baseConfig)
+
+  val testKit = new TestKit(ActorSystem("MarketActorBenchmark", appConfig))
+
+  val prng = new Random(appConfig.getLong("market-actor-benchmark.seed"))
+
+  /* Setup the tradables. */
+  val numberTradables = appConfig.getInt("market-actor-benchmark.tradables.number")
+  val tradables = for (i <- 1 to numberTradables) yield {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use braces in for comprehensions](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2424830928/issues/source?bid=2921361&fileBranchId=3239634#l34)